package.json

```diff
@@ -99,9 +100,11 @@
   "devDependencies": {
     "@netlify/functions": "^2.6.0",
     "@netlify/plugin-nextjs": "^4.41.3",
+    "@types/lowlight": "^0.0.7",
```

Comment:
`@types/*` needs to be installed to `dependencies` instead of `devDependencies` otherwise the Netlify Build would fail.